src/selenium/basePage.js

```diff
@@ -0,0 +1,18 @@
+var until = require('selenium-webdriver').until;
+var By = require('selenium-webdriver').By;
+
+function BasePage(webdriver) {
+  this.driver = webdriver;
+}
+
+BasePage.prototype.visit = function(url) {
```

Comment:
@geekyd Thanks for the review :). You are right. I used the hackish trick of emulating class behavior in ES5. I then read about the class introduced in the ES6 and they are nothing but an abstraction over the actual prototypical inheritance Javascript provides. I just revised about the prototypical inheritance and its advantage over the class inheritance. There is no tight coupling and inflexible hierarchy problem. Also Class doesn't really exist in JS and they were just introduced for syntactic sugar and to reduce the wrong hackish behavior to emulate it (like I did :grinning:) and provide that as a built-in feature. 

So, I redid the thing using prototypical inheritance here. The code is much cleaner now. Please review again. Thanks!!